src/bin/vip.js

```diff
@@ -151,13 +149,12 @@ const rootCmd = async function() {
 		}
 
 		try {
-			Token.set( token.raw );
+			await Token.set( token.raw );
 		} catch ( err ) {
```

Comment:
If we don't `await` a Promise, no errors it might throw will be caught by `catch`.

The alternative is `Token.set( token.raw ).catch( ( err ) => { /* ... */ } );